File: Backend/src/main/java/com/example/project/dto/request/PickupPointRequest.java
package com.example.project.dto.request;

import com.fasterxml.jackson.annotation.JsonProperty;
import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.NotNull;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;


@AllArgsConstructor
@NoArgsConstructor
@Data
public class PickupPointRequest {
    @JsonProperty("manager_id")
    @NotNull(message = "ID менеджера не может быть пустым")
    Integer managerId;

    @NotBlank(message = "Адрес не может быть пустым")
    String address;
}

File: Android/app/src/main/java/com/kire/market_place_android/data/remote/dto/request/admin/PickUpPointRequest.kt
package com.kire.market_place_android.data.remote.dto.request.admin

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class PickUpPointRequest(
    @SerialName("manager_id")
    val managerId: Int = 0,
    val address: String = ""
)
